update logo for both github themes

ahaWDY opened the pull request:
Signed-off-by: ahaWDY <2503131624@qq.com>

<!--
Thank you for your Pull Request. Please provide a description and review
the requirements below.

Contributors guide: https://github.com/theia-ide/theia/blob/master/CONTRIBUTING.md
-->

#### What it does
Fixes #9114 

#### How to test
Preview the updated logo.

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

vince-fugnitto commented:
@ahaWDY after further discussion, we decided on not using blue as to not confuse with the _Eclipse Theia Blueprint_ (an open-source product built with the framework). Instead, can we use gray `#9C9C9C`?

ahaWDY commented:
> @ahaWDY after further discussion, we decided on not using blue as to not confuse with the _Eclipse Theia Blueprint_ (an open-source product built with the framework). Instead, can we use gray `#9C9C9C`?

Alright, I have changed it to gray `#9C9C9C`.

vince-fugnitto commented:
> Alright, I have changed it to gray `#9C9C9C`.

Thank you very much 👍 Please squash and I’ll merge the changes.

ahaWDY commented:
> > Alright, I have changed it to gray `#9C9C9C`.
> 
> Thank you very much 👍 Please squash and I’ll merge the changes.

I have squashed! Thank you for your patience!

vince-fugnitto reviewed (approved):
@ahaWDY thank you for your first contribution! 👍
